### PR TITLE
Intrinsic & extrinsic sizing rename 'fill-available' to 'fill'

### PR DESCRIPTION
**Pull request by yisibl**
https://drafts.csswg.org/css-sizing/?c=M%3BO%3DD#valdef-width-fill

Before:

```css
a {
    width: fill-available;
}
```

After:

```css
a {
    width: fill;
}
```

### CODE REVIEW

**Comment by ai** [comment]
@yisibl Thanks for remind. You are the best with W3C specs :)

**Comment by yisibl** [comment]
@ai Thanks :ghost: . 

What time are you going to merge?

**Comment by ai** [comment]
Definitely. I just need to add support for old value with a warning.

**Comment by ai** [comment]
Done dd6359d

File: data/prefixes.coffee [hunk 1]
```diff
@@ -235,9 +235,15 @@ feature require('caniuse-db/features-json/css3-tabsize'), (browsers) ->
 
 # Intrinsic & extrinsic sizing
 feature require('caniuse-db/features-json/intrinsic-width'), (browsers) ->
-  prefix 'max-content', 'min-content', 'fit-content', 'fill-available',
+  # Rename 'fill-available' to 'fill'
+  # https://drafts.csswg.org/css-sizing/?c=M%3BO%3DD#valdef-width-fill
```
Comment (yisibl) [inline data/prefixes.coffee]: You can use clean url:
https://drafts.csswg.org/css-sizing/#valdef-width-fill

Comment (ai) [inline data/prefixes.coffee]: I removed all comments :D. I think they will not be useful for other developers

**Pull request merged**